src/Executables/Game/World/Entities/PlayerEntity.cs

```diff
@@ -269,6 +270,26 @@ private void CalculateDefence()
             // todo add defence bonus from quests
         }
 
+        private void CalculateMovement()
+        {
+            MovementSpeed = 150;
+            float modifier = 0;
+            foreach (var slot in Enum.GetValues<EquipmentSlots>())
+            {
+                var item = Inventory.EquipmentWindow.GetItem(slot);
+                if (item == null) continue;
+                var proto = _itemManager.GetItem(item.ItemId);
+                if (proto?.Type != (byte)EItemType.Armor) continue;
+                modifier += (byte)proto.Applies[0].Value;
+            }
+            if(modifier != 0)
+            {
+                MovementSpeed = (byte) (MovementSpeed * (1+(modifier/100)));
```

Comment:
Also since `MovementSpeed` is a byte, it might be a good idea to limit it with a maximum value to prevent overflow.

Reply:
The `if(modifier != 0)` unnecessary here.

Reply:
Thanks, both have been added/changed